static/js/components/ShiftManagement.jsx

```diff
@@ -140,23 +148,12 @@ const userLabel = (user) => {
   return label;
 };
 
-function CurrentShiftMenu() {
+function CurrentShiftMenu({ currentShift }) {
   const classes = useStyles();
-  const { shiftList } = useSelector((state) => state.shifts);
   const { permissions } = useSelector((state) => state.profile);
-  const { currentShift } = useSelector((state) => state.shift);
   const currentUser = useSelector((state) => state.profile);
```

Comment:
I think you may be grabbing the profile twice (just get the permissions from the current user.